<launch>

    <!--Set params and launch relative measurement generator-->
    <include file="$(find relative_meas_gen)/launch/relative_measurement_generator.launch">
        <arg name="robot_ID" value="2"/>
        <arg name="host_robot" value="robot2"/>
        <arg name="camera_ID" value="camera2"/>
        <arg name="tagsize" value=".17"/>
        <arg name="measurement_topic" value="/relative_inter_robot_meas"/>
        <arg name="images_topic" value="/race1/main_camera/image_raw/compressed"/>
        <arg name="config_file" value="$(find relative_meas_gen)/params/CoPeD.yaml"/>
        <arg name="base_link" value="wilbur/imu_link"/>
        <arg name="camera_link" value="wilbur/stereo_left_optical_frame"/>
    </include>
</launch>